Source/Views/ActionSheetCancelActionView.swift

```diff
@@ -73,6 +73,7 @@ final class ActionSheetCancelActionView: UIView {
             // underneath the label to avoid blurring the label. The blur passes through touches so the button
             // can still be tapped.
             self.bringSubviewToFront(self.blurBackground)
+            if visualStyle.backgroundColor != nil { self.blurBackground.effect = nil }
```

Comment:
If we're not going to set an effect if there's a background color I believe adding the background blur altogether is kinda pointless, so let's move this up to the `buildView` method and not even call `addBlurBackground` if a background color is set.

Similar to the condition from above, it would probably have to check both `visualStyle.backgroundColor` and `visualStyle.actionViewCancelBackgroundColor`.